### PR TITLE
Updated main README and supported_examples with MMQA details

### PR DESCRIPTION
**Pull request by okhleif-10**
## Description

Added more information about MultimodalQnA use case in `README.md` and `supported_examples.md`

## Issues

N/A

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

N/A

### CODE REVIEW

File: supported_examples.md [hunk 1]
```diff
@@ -186,7 +185,16 @@ FAQ Generation Application leverages the power of large language models (LLMs) t
 
 ### MultimodalQnA
 
-[MultimodalQnA](./MultimodalQnA/README.md) addresses your questions by dynamically fetching the most pertinent multimodal information (frames, transcripts, and/or captions) from your collection of videos.
+[MultimodalQnA](./MultimodalQnA/README.md) addresses your questions by dynamically fetching the most pertinent multimodal information (frames, transcripts, and/or captions) from your collection of videos or images. MultimodalQnA utilizes BridgeTower model, a multimodal encoding transformer model which merges visual and textual data into a unified semantic space. During the video ingestion phase, the BridgeTower model embeds both visual cues and auditory facts as texts, and those embeddings are then stored in a vector database. When it comes to answering a question, the MultimodalQnA will fetch its most relevant multimodal content from the vector store and feed it into a downstream Large Vision-Language Model (LVM) as input context to generate a response for the user.
```
Comment (dmsuehir) [inline supported_examples.md]:
```suggestion
[MultimodalQnA](./MultimodalQnA/README.md) addresses your questions by dynamically fetching the most pertinent multimodal information (frames, transcripts, and/or captions) from your collection of videos, images, or audio files. MultimodalQnA utilizes BridgeTower model, a multimodal encoding transformer model which merges visual and textual data into a unified semantic space. During the ingestion phase, the BridgeTower model embeds both visual cues and auditory facts as texts, and those embeddings are then stored in a vector database. When it comes to answering a question, the MultimodalQnA will fetch its most relevant multimodal content from the vector store and feed it into a downstream Large Vision-Language Model (LVM) as input context to generate a response for the user.
```

File: supported_examples.md [hunk 2]
```diff
@@ -186,7 +185,16 @@ FAQ Generation Application leverages the power of large language models (LLMs) t
...
+
+| Service   | Model                                                                               | HW   | Description              |
+| --------- | ----------------------------------------------------------------------------------- | ---- | ------------------------ |
+| Embedding | [BridgeTower/bridgetower-large-itm-mlm-itc](https://huggingface.co/BridgeTower/bridgetower-large-itm-mlm-itc) | Xeon/Gaudi | Video embeddings service |
+| Embedding | [BridgeTower/bridgetower-large-itm-mlm-gaudi](https://huggingface.co/BridgeTower/bridgetower-large-itm-mlm-gaudi) | Gaudi | Video embeddings service |
+| LVM       | [llava-hf/llava-1.5-7b-hf](https://huggingface.co/llava-hf/llava-1.5-7b-hf)        | Xeon/Gaudi | LVM service              |
```
Comment (dmsuehir) [inline supported_examples.md]: Gaudi doesn't support llava 1.5 because it's using TGI-gaudi, which only supports 1.6

File: supported_examples.md [hunk 3]
```diff
@@ -186,7 +185,16 @@ FAQ Generation Application leverages the power of large language models (LLMs) t
...
+
+| Service   | Model                                                                               | HW   | Description              |
+| --------- | ----------------------------------------------------------------------------------- | ---- | ------------------------ |
+| Embedding | [BridgeTower/bridgetower-large-itm-mlm-itc](https://huggingface.co/BridgeTower/bridgetower-large-itm-mlm-itc) | Xeon/Gaudi | Video embeddings service |
```
Comment (dmsuehir) [inline supported_examples.md]: Maybe "Video embedding service" ➡️  "Multimodal embedding service"?

File: supported_examples.md [hunk 4]
```diff
@@ -186,7 +185,16 @@ FAQ Generation Application leverages the power of large language models (LLMs) t
...
+| Embedding | [BridgeTower/bridgetower-large-itm-mlm-gaudi](https://huggingface.co/BridgeTower/bridgetower-large-itm-mlm-gaudi) | Gaudi | Video embeddings service |
+| LVM       | [llava-hf/llava-1.5-7b-hf](https://huggingface.co/llava-hf/llava-1.5-7b-hf)        | Xeon/Gaudi | LVM service              |
+| LVM       | [llava-hf/llava-1.5-13b-hf](https://huggingface.co/llava-hf/llava-1.5-13b-hf)        | Xeon/Gaudi | LVM service              |
+| LVM       | [llava-hf/llava-v1.6-vicuna-13b-hf](https://huggingface.co/llava-hf/llava-v1.6-vicuna-13b-hf)        | Xeon/Gaudi | LVM service              |
```
Comment (dmsuehir) [inline supported_examples.md]: I haven't successfully been able to run 1.6 vicuna on Xeon, but this is the default model used for Gaudi

**Review by dmsuehir — CHANGES_REQUESTED** [review]

**Review by dmsuehir — APPROVED** [review]
Thanks LGTM

**Review by mhbuehler — APPROVED** [review]
LGTM

**Pull request merged**